### ISSUE TITLE
onGet not working anymore as expected

### ISSUE THREAD

**Issue by Regaddi** [ctimmerm/axios-mock-adapter#41]
Hi @ctimmerm,

it seems, that 1.8.0 broke the behaviour of `onGet`.
Before all GET requests matching my URL defined in `onGet` landed the reply callback function.

Now I have to explicitly define EVERY SINGLE parameter, to have that match.

Example:

`GET /users?q=Adam`

```
mock.onGet('/users').reply(conf => {
  // not reached!!
});
mock.onGet('/users', { params: { q: 'Adam' }}).reply(conf => {
  // reached
});
```

**Comment by ctimmerm** [comment]
Thanks for the report, I'll check it out.

**Comment by ctimmerm** [comment]
I'm not the original author of this feature, but it seems the current implementation was intentional, as this test shows: https://github.com/ctimmerm/axios-mock-adapter/blob/master/test/basics.spec.js#L127-L133

However, I agree that this change is unexpected and causes test failures for many people.
I will soon release v1.8.1 that restores the old behavior.

Sorry for the inconvenience this has caused.

**Comment by ctimmerm** [comment]
Should be fixed in 08ea410c14bcf2f6b6a51be378d89fab18eff739, released as 1.8.1

**Comment by Regaddi** [comment]
Wonderful! Thanks!
